### ISSUE TITLE
Moneromooo-monero

### ISSUE THREAD

**Issue by MoneyAS** [monero-project/monero-site#636]
Could you get the Moneromooo-moreno (Moreno Wallet Generator) to answer my Issue.

**Comment by QuickBASIC** [comment]
I assume you mean moneromooo-monero/monero-wallet-generator#20 ?

This is the incorrect place to ask for assistance for this issue. If you're trying to contact someone directly, you might try on #monero on freenode IRC.

This is the second time you've opened an issue for this reason. Please do not open any further issues on this repo unless it has to do with the Monero website (getmonero.org).

+invalid
+wontfix

@luigi1111  Can we consider blocking @MoneyAS from the Organization Settings on GitHub if he opens any further issues?

**Comment by QuickBASIC** [comment]
+resolved
